test/languageService.test.ts

```diff
@@ -0,0 +1,70 @@
+import DecompressProvider from "../src/languageservice/decompressProvider";
+import { IPackage, IStatusView } from "../src/languageservice/interfaces";
+import { ILogger } from "../src/models/interfaces";
+import { assert } from "chai";
+import HttpClient, { IDownloadProgress } from "../src/languageservice/httpClient";
+import { HttpModule } from "../out/src/views/htmlcontent/src/js/lib/@angular/http";
+
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+suite('Language Service Tests', () => {
+
+    suite('Decompress Provider Tests', () => {
+        let decompressProvider = new DecompressProvider();
+
+        test('Decompress package test', async () => {
+            let testPackage: IPackage = {
+                url: 'test_url',
+                tmpFile: undefined
+            };
+            let testLogger: ILogger = {
+                logDebug: undefined,
+                increaseIndent: undefined,
+                decreaseIndent: undefined,
+                append: undefined,
+                appendLine: undefined
+
+            };
+            try {
+                await decompressProvider.decompress(testPackage, testLogger);
```

Comment:
You need to fail the test after the away - otherwise even if an error isn't throw the test will pass